exampleSite/hugo.yaml

```diff
@@ -86,6 +86,7 @@ menu:
 
 params:
   description: Modern, responsive, batteries-included Hugo theme for creating beautiful static websites.
+  fullWidth: false
```

Comment:
can you revert this

Reply:
Oh sorry! I didn't mean to include that. I was messing around with the full-width issue.

Reply:
also i saw there are some empty line changes, can you revert those as well?